Disable video logo as this breaks cross-building, which runs bmp_logo
during the build to generate the logo headers. Debian does not
currently ship the bmp_logo tool, so simply disable this feature.

Index: u-boot/configs/tools-only_defconfig
===================================================================
--- u-boot.orig/configs/tools-only_defconfig
+++ u-boot/configs/tools-only_defconfig
@@ -28,6 +28,7 @@ CONFIG_SOUND=y
 CONFIG_SYSRESET=y
 CONFIG_TIMER=y
 CONFIG_VIDEO=y
+# CONFIG_VIDEO_LOGO is not set
 # CONFIG_VIRTIO_MMIO is not set
 # CONFIG_VIRTIO_PCI is not set
 # CONFIG_VIRTIO_SANDBOX is not set
